wiki/osu!stream/de.md

```diff
@@ -0,0 +1,375 @@
+
+Die Hintergrundfarbe ist violett und die Lichter verdunkeln sich, abhängig von den aktuellen Lebenspunkten
+
+## Benutzeroberfläche
+
+---
+
+![](img/Os-MM.png "Hauptmenü")
+
+---
+
+![](img/Os-SL.jpg "Songliste")
+
+---
+
+![](img/Os-Store.jpg "Store")
+
+---
+
+![](img/Os-DS.png "Wahl des Schwierigkeitsgrad")
+
+---
+
+![](img/Os-SI.png "Songinformationen")
+
+---
+
+![](img/Os-PlayExpert.jpg "Expertenmodus")
+
+---
+
+![](img/Os-Pause.png "Pausenmenü")
+
+---
+
+![](img/Os-FM.png "Menü bei einem gescheiterten Versuch")
+
+---
+
+![](img/Os-SC.png "Stage cleared!")
+
+---
+
+![](img/Os-Grade.jpg "Ergenisbildschirm")
+
+---
+
+![](img/Os-O1.png "Optionsmenü 1")
+
+---
+
+![](img/Os-O2.png "Optionsmenü 2")
+
+---
+
+## Songliste
+
+***[Klicke hier für die gesamte Liste](Song_List)***
+
+## Punktevergabe
+
+*Hinweis: Die Rangliste kann noch im Spiel angesehen werden, allerdings werden neue Punktestände angenommen.*
+
+### Genauigkeit
+
+`Genauigkeit = Trefferwert / (perfekter Trefferwert)`
+
+| Begriff | Formel |
+| :-: | :-- |
+| **Trefferwert** | (Anzahl der Misses \* 0 + Anzahl der 50er \* 1 + Anzahl der 100er * 2 + Anazhl der 300er \* 4) |
+| **perfekter Trefferwert** | (Anzahl der Misses + Anzahl der 50er + Anzahl der 100er + Anzahl der 300er) \* 4 |
+
+Mit anderen Worten, jeder Miss ist 0%, jeder 50er ist 25%, jeder 100er ist 50% und jeder 300er ist 100% Genauigkeit wert.
+
+### Punkte
+
+Die Punkte, die von jedem Hit-Circle und jedem Ende eines Sliders vergeben werden, wird mit folgender Formel berechnet:
+
+`Punkte = Trefferwert + Trefferwert * (Combomultiplizierer * fester Multiplizierer) / 25`
+
+| Begriff | Bedeutung |
+| :-: | :-- |
+| **Trefferwert** | Die Bewertung des Hit-Circles (50, 100 oder 300). |
+| **Combomultiplizierer** | (Combo vor dem Treffer - 1) oder 0; abhängig davon, was höher ist. |
+| **Fester Multiplizierer** | Der Multiplizierer der den Punktestand bei 600.000 fixiert. |
+
+Zusätzlich gibt jeder Sliderstart, -ende und -repeat tick 30 Punkte und jede volle Umdrehung eines Spinners gibt 100 Punkte.
+Es gibt einen weiteren Bonus von 10 Punkten, abhängig von den Umdrehungen pro Minute nachdem die Spinnermessbar gefüllt ist.
+
+### Bewertung des Hit-Circles
+
+**Hit-Circles**
+
+- Eine 300, 100 oder 50 wird bei einem normalen Hit-Circle verliehen, abhängig von der Treffgenauigkeit.
+- Ein Miss wird vergeben, wenn ein Hit-Circle zu früh oder überhaupt nicht getroffen wurde.
+  - Wenn das Objekt viel viel zu früh getroffen wird, wackelt es stattdessen. Nichts wird ansosnten passieren, außer die Visualisierung, dass das Objekt zu früh getroffen wurde.
+
+**Hold-Circle**
+
+- Die Bewertung des Hold-Circles hängt ab von dem initialen Treffer und die Haltezeit, bis der Kreis explodiert.
+  - 300 bei perfekter Ausführung.
+  - 100 für einen leicht inkorrekten initialen Treffer und den Kreis richtig halten.
```

Comment:
```suggestion
  - 100 für einen leicht inkorrekten initialen Treffer und das richtige Halten des Kreises.
```

Reply:
It needs to be `dem richtigen Halten des Kreises`, doesn't it?